### DISCUSSION TITLE
Will pubsubhubbub be supported in V3? Or will the old one be deprecated?

### DISCUSSION THREAD

**Discussion by thedewpoint** [etsy/open-api#679] [Q&A]
I'm curious if we can rely on the pubsubhubbub documentation or if this will be retired?

**Comment by thedewpoint** [comment]
bump @etsyachristensen do you know by any chance

**Reply by etsyachristensen** [reply]
I don't know the fate of pubsubhubbub, but I can say that no work has gone into updating it. I'd suspect it will go away as well. We have been internally discussing webhooks and planning to roll that out at some point. I have no expectation on when that might be.

**Reply by JoannSellerServices** [reply]
Is there any planned solution for feed updates in v3 or workarounds until webhooks are released?  Pubhub is a core aspect of our application, so just wondering how we might circumvent this if it is deprecated.

**Reply by JoannSellerServices** [reply]
@nateswart Anyone on the Etsy team able to provide any additional detail / scope around possible webhooks availability and what they will cover?

**Comment by sampok** [comment]
We are also relying on the v2 real-time feed. Instead of PubSubHubbub we're using [the /listings/latest endpoint](https://www.etsy.com/developers/documentation/getting_started/real_time_feeds).

Waiting for news on [v3 real-time feed](https://github.com/etsy/open-api/issues/75) or some type of a [workaround solution](https://github.com/etsy/open-api/issues/528).

**Reply by benjymouse** [reply]
We started using this. Very useful. I hope this is continued in v3.

**Comment by JoannSellerServices** [comment]
update from the Etsy team via direct email on this topic:

"I hope this email finds you well. Thanks for getting in touch with us about your concern. 

At this time we do not have web hooks or any other way to notify you of a new order in realtime. The best way to get this information is by periodically polling our API to check for new orders. Be sure to pick a frequency that will keep your app within your rate limits.

I do understand that this is not the most efficient approach and will pass on your desire for web hooks to my team. We are always looking for ways to improve our Open API.

I hope this helps. Please let me know if you have further questions."

**Reply by Gareth064** [reply]
They have been saying that for two years. Even though there has been lots of support for wrbhooks etc, they still ain't doing it.

**Comment by FloderCC** [comment]
Hello, any update? Thanks in advance
